File: easy/go/0026.go
func removeDuplicates(nums []int) int {
    if len(nums) == 0{
        return 0
    }
    var head int = 1
    var prev int = nums[0]
    var size int = len(nums)
    for i := 0 ; i < size ; i++{
        if nums[i] != prev{
            nums[head] = nums[i]
            prev = nums[i]
            head += 1
        }
    }
    return head
}

File: easy/c/0026.c
int removeDuplicates(int* nums, int numsSize){
    if(numsSize<=1) return numsSize;
    int head = 1;
    int prev = nums[0];
    for(int i = 1; i < numsSize; ++i){
        if(nums[i] != prev){
            prev = nums[i];
            nums[head++] = nums[i];
        }
    }
    return head;
}